lib/eigensolve_quda.cpp

```diff
@@ -203,19 +217,15 @@ namespace quda
     double eps = 0.0;
     switch (prec) {
     case QUDA_DOUBLE_PRECISION:
-      if (getVerbosity() >= QUDA_SUMMARIZE) printfQuda("Running Eigensolver in double precision\n");
```

Comment:
I think these `printf`s are useful, but switch to `QUDA_VERBOSE` instead of `QUDA_SUMMARIZE`.

Reply:
I removed the printf, it was just causing clutter from multiple calls to `setEpsilon`.